### DISCUSSION TITLE
תשובה לסעיף ו' מבחן 2020 מועד א'

### DISCUSSION THREAD

**Discussion by micha-sherman** [NuclearGandhi/technion_second_brain#11] [טעויות ותיקונים]
אני חושב שמה שרוצים מאיתנו זה שנכפול את הכוח הממוצע שחישבנו קודם ברדיוס החדש ונקבל את המומנט הנחוץ לסיבוב.
אני איתך בתסכול, קורא את ההערות צוחק, קצת בוכה, מנגב את הדמעות וממשיך לפתור...

**Comment by NuclearGandhi** [comment]
וואלה יכול להיות, קשה לוודא בלי פתרונות רשמיים
